curator/views/study/edit.html

```diff
@@ -513,7 +513,7 @@
                 </div>
               </div>
 
-              <div class="control-group">
+              <!-- <div class="control-group">
```

Comment:
Just a reminder that the `^ot:notIntendedForSynthesis` flag does have a minor effect in the curation app. If it's checked, we relax the "Study quality" scoring by skipping (passing) these tests:
  - For studies nominated for synthesis, there should be at least one preferred tree.
  - Preferred trees should not have duplicate (non-monophyletic) tips mapped to a single taxon.
  - All tip labels in preferred trees should be mapped to the Open Tree Taxonomy.

Without this checkbox, all studies will be subject to more stringent tests by default.